pkgs/top-level/python-packages.nix

```diff
@@ -29288,4 +29288,47 @@ in modules // {
       sha256 = "14kb9gxw39zhvrijhp066b4bm6bgv35iw56c394y4dyczpha0dij";
     };
   };
+
+  sqlalchemy_bb8 = buildPythonPackage rec {
```

Comment:
To be clear, living in python-packages mostly buys you convention and some machinery to magically appear for all compatible versions of python (2.7, 3.x, etc.). If you don't need that for BB, it makes more sense to put it with your expression.